### ISSUE TITLE
#keepTileSprite on tile doesn't work?

### ISSUE THREAD

**Issue by 19lmyers** [Picalines/godot-ldtk-import-mono#7]
I have an example project demonstrating the bug, using a LDtk template edited to add a ladder tile entity.
The problem, to be specific, is that the importer isn't finding the Sprite and setting its texture.

**Comment by 19lmyers** [comment]
[testproject.zip](https://github.com/Picalines/godot-ldtk-import-mono/files/9274176/testproject.zip)

**Comment by Picalines** [comment]
I will try to fix it during the day. Thank you for all the reports

**Comment by Picalines** [comment]
The bug is that Godot does not save the change of the child sprite to the scene file.
At first I thought it was easy to fix this by setting the Owner field of the sprite node, but that ends up making a copy rather than changing the original sprite.

Godot has "Editable children" checkbox in the context menu, but there's no way of accessing it in code. I found mentions of the `set_editable_instance` method on the forum, but it does not seem to be in the current version of the engine. It sounds like a perfect solution.

Honestly i don't know how to fix it right now. Bad I didn't test it first :/

There are `#tileId` and `#tileSrc` fields for tile entities, it's possible to write a code that assigns correct sprite region. `#keepTileSprite` was intended as a shortcut

**Comment by 19lmyers** [comment]
Yeah, I'm good with using #tileSrc as a quick fix. I had a feeling this didn't have an easy solution.
